src/views/visual/visual-view-graph.js

```diff
@@ -1,43 +1,105 @@
-import { css, customElement, html, LitElement, property } from 'lit-element';
+import { customElement, html, LitElement, property } from 'lit-element';
 
 import { getDataForVisual } from '../../api/actions';
 import '../utility/LoadingSpinner';
 import { getGoogleGraphOptions } from './visualViewUtils';
 
+import styles from './visual-view-graph.styles';
+
 @customElement('visual-view-graph')
 export class VisualViewGraph extends LitElement {
-  @property({ type: Array }) graphData;
-  @property({ type: String }) language;
   @property({ type: String }) searchItem;
   @property({ type: String }) selectedCollections;
+  @property({ type: Number }) pageSize;
+  @property({ type: Function }) setSelection;
+
+  @property({ type: Array }) graphData;
+  @property({ type: String }) language;
+  @property({ type: Number }) currentPage = 0;
+  @property({ type: Number }) totalPages;
   @property({ type: String }) chartHeight;
   @property({ type: String }) fetchLoading;
   @property({ type: String }) fetchError;
 
   static get styles() {
-    return [
-      css`
-        #parallels-chart {
-          font: 13px sans-serif;
-          padding: 30px;
-          width: 93vw;
-        }
-      `,
-    ];
+    return [styles];
   }
 
   updated(_changedProperties) {
-    console.log('visual view updated. ', _changedProperties);
     _changedProperties.forEach((oldValue, propName) => {
       if (
         ['searchItem', 'selectedCollections'].includes(propName) &&
         !this.fetchLoading
       ) {
         this.fetchData();
       }
+
+      if (['currentPage'].includes(propName) && !this.fetchLoading) {
```

Comment:
if we're comparing against one name, I think `propName === "currentPage"` would be more readable.